README.md

```diff
@@ -18,6 +18,31 @@ npm install ng-block-ui --save
 npm install ng-block-ui@next --save
 ```
 
+### Configuring SystemJS
+
+If your project is using SystemJS for module loading, you will need to add ng-block-ui to the SystemJS configuration:
+
+```js
+System.config({
+  // existing configuration options
+  map: {
+    ...
+    'ng-block-ui': 'npm:ng-block-ui/dist',
```

Comment:
@sachinvettithanam, this just needs to be `npm:ng-block-ui/dist/umd` (see #8 for more info) and then this LGTM :+1: